src/App.js

```diff
@@ -50,6 +55,8 @@ function App() {
     <Container>
       <ToggleEdit
         edit={edit}
+        error={formik.touched.task && Boolean(formik.errors.task)}
+        helperText={formik.touched.task && formik.errors.task}
```

Comment:
why same thing twice ?

Reply:
error  shows  input turns red and helpertext it show required

Reply:
why not just pass in `formik.touched.task && formik.errors.task`, please read